broadinstitute/pearl
# JN-478 updated admin ui header

This adds a help link in the top right, and replaces the user name with a user icon.  Goal is to move towards the invision prototype here https://dspuxteam.invisionapp.com/overview/Juniper---Nav-Concept-clk06hf7d2iiw01ah8a517ir0/screens?utm_source=email&sortBy=1&sortOrder=1&viewLayout=2

Nav look
![image](https://github.com/broadinstitute/juniper/assets/2800795/9b09cb25-2c8e-4c2f-b67f-f1d4a33b3767)

Help popup
![image](https://github.com/broadinstitute/juniper/assets/2800795/166f56b0-550e-43ff-b5ea-2b3b32b19e9d)

TO TEST:
1. go to admin tool
2. click around the top-right menu, confirm stuff still works
